src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Update AWSSDK dependencies to v4.
```

Comment:
IMO This should be prefixed with `BREAKING: ` as v4 is not backwards compatible with v3